tests/Tests.AzureAppConfiguration/FeatureManagementTests.cs

```diff
@@ -172,6 +172,9 @@ public void WatchesFeatureFlags()
                 eTag: new ETag("c3c231fd-39a0-4cb6-3237-4614474b92c1" + "f"));
 
             featureFlags.Add(_kv2);
+
+            // Sleep to let the cache expire
+            Thread.Sleep(1000);
```

Comment:
The sleep here looks like it's necessary since we are waiting for the cache to expire. But can we update the test to use the same timespan that we pass into `options.UseFeatureFlags(o => o.CacheExpirationInterval` so they don't get out of sync.

Reply:
After I saw that you added this sleep I was wondering to myself, how did this ever pass in the first place? Then I realize it was only passing because of a bug